.gitmodules

```diff
@@ -13,3 +13,6 @@
 [submodule "solidity_contracts/lib/base64-sol"]
 	path = solidity_contracts/lib/base64-sol
 	url = https://github.com/Brechtpd/base64
+[submodule "solidity_contracts/lib/openzeppelin-contracts"]
+	path = solidity_contracts/lib/openzeppelin-contracts
+	url = https://github.com/OpenZeppelin/openzeppelin-contracts
```

Comment:
already un submodules

Reply:
Yes there are used in the tests for uniswapv3. The version of the library used for the uniswapv3 is 3.4 which if upgraded to 5.0 will bring breaking changes leading to code modifications to uniswapv3
My suggestion here is to keep both. 
Other solution would be to delete the uniswapv3 test and code as it is skipped

Reply:
or make the submodule name explicit

Reply:
Done 
Should be better